src/Resources/config/no_driver_admin.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+</container>
```

Comment:
?

Reply:
This file is loaded from [SonataMediaExtension.php](https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/DependencyInjection/SonataMediaExtension.php#L147) when SonataAdminBundle is enabled. So, i created an empty file for that case.

Reply:
Please add an XML comment to explain this.